freqtrade/exchange/binance.py

```diff
@@ -32,13 +32,23 @@ def get_order_book(self, pair: str, limit: int = 100) -> dict:
 
         return super().get_order_book(pair, limit)
 
-    def stoploss_limit(self, pair: str, amount: float, stop_price: float, rate: float) -> Dict:
+    def stoploss_adjust(self, stop_loss: float, order: Dict) -> bool:
+        """
+        Verify stop_loss against stoploss-order value (limit or price)
+        Returns True if adjustment is necessary.
+        """
+        return order['type'] == 'stop_loss_limit' and stop_loss > float(order['info']['stopPrice'])
+
+    def stoploss(self, pair: str, amount: float, stop_price: float, order_types: Dict) -> Dict:
         """
         creates a stoploss limit order.
         this stoploss-limit is binance-specific.
         It may work with a limited number of other exchanges, but this has not been tested yet.
-
         """
+        # Limit price threshold: As limit price should always be below stop-price
+        LIMIT_PRICE_PCT = order_types.get('stoploss_on_exchange_limit_ratio', 0.99)
+        rate = stop_price * LIMIT_PRICE_PCT
```

Comment:
I don't like this in the capital letters (`LIMIT_PRICE_PCT`), maybe simple `limit_price_pct` is better...
(I noticed where you copied this from, but let's clean this up...)